src/content/drafts/eslint-config-inspector.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: Introducing ESLint Config Inspector
+teaser: "Introducing the ESLint Config Inspector, a visual tool to help you understand and inspect ESLint flat configuration files."
+tags:
+  - Config Inspector
+  - Release
+authors:
+  - antfu
+categories:
+  - Announcements
+---
+
+As the [new flat config format](/blog/2023/10/flat-config-rollout-plans/) rolling out with many benefits, it makes configures easier to manage and more transparent to compose. However, it can still be non-trivial to understand what rules are enabled and disabled for which files, especially when the configurations are complex or extending from multiple sources.
+
+To help with that situation and improve the developer experience with flat configs even further, we are excited to introduce you the [ESLint Config Inspector](https://github.com/eslint/config-inspector) --- a visual and interactive tool helping you to understand and inspect your config file.
```

Comment:
Cleaning up some grammar.
```suggestion
As of ESLint v9,0.0, the [new configuration system](/blog/2023/10/flat-config-rollout-plans/) has reached general availability, bringing with it with many benefits. Configuration files are now easier to manage and more transparent to compose. However, it can still be non-trivial to understand which rules are enabled and disabled for specific files, especially when your configuration is complex or composed from multiple sources. That's why we are excited to introduce the [ESLint Config Inspector](https://github.com/eslint/config-inspector), a visual and interactive tool to help you better understand and inspect your config file.
```